beacon_node/beacon_chain/src/light_client_server_cache.rs

```diff
@@ -312,6 +370,96 @@ impl<T: BeaconChainTypes> LightClientServerCache<T> {
     pub fn get_latest_optimistic_update(&self) -> Option<LightClientOptimisticUpdate<T::EthSpec>> {
         self.latest_optimistic_update.read().clone()
     }
+
+    pub fn get_sync_committee_branch(
+        &self,
+        store: &BeaconStore<T>,
+        block_root: &Hash256,
+    ) -> Result<Option<FixedVector<Hash256, CurrentSyncCommitteeProofLen>>, BeaconChainError> {
+        let column = DBColumn::SyncCommitteeBranch;
+
+        if let Some(bytes) = store
+            .hot_db
+            .get_bytes(column.into(), &block_root.as_ssz_bytes())?
+        {
+            // TODO unwrap
+            let sync_committee_branch: FixedVector<Hash256, CurrentSyncCommitteeProofLen> =
+                FixedVector::from_ssz_bytes(&bytes).unwrap();
+
+            return Ok(Some(sync_committee_branch));
+        }
+
+        Ok(None)
+    }
+
+    pub fn get_sync_committee(
+        &self,
+        store: &BeaconStore<T>,
+        sync_committee_period: u64,
+    ) -> Result<Option<SyncCommittee<T::EthSpec>>, BeaconChainError> {
+        let column = DBColumn::SyncCommittee;
+
+        if let Some(bytes) = store
+            .hot_db
+            .get_bytes(column.into(), &sync_committee_period.as_ssz_bytes())?
+        {
+            let sync_committee: SyncCommittee<T::EthSpec> =
+                SyncCommittee::from_ssz_bytes(&bytes).unwrap();
+            return Ok(Some(sync_committee));
+        }
+
+        Ok(None)
+    }
+
+    #[allow(clippy::type_complexity)]
+    pub fn get_light_client_bootstrap(
+        &self,
+        store: &BeaconStore<T>,
+        block_root: &Hash256,
+        finalized_period: u64,
+        chain_spec: &ChainSpec,
+    ) -> Result<Option<(LightClientBootstrap<T::EthSpec>, ForkName)>, BeaconChainError> {
+        if let Some(block) = store.get_full_block(block_root)? {
```

Comment:
```suggestion
        let Some(block) = store.get_full_block(block_root)? else { return None };
```
to un-indent everything